crates/turbopack-ecmascript/src/parse.rs

```diff
@@ -376,8 +397,9 @@ async fn parse_content(
                     .await?;
             }
 
+            // seems already emitted errors are not readable except the count?
```

Comment:
You need to implement [`Emitter`](https://rustdoc.swc.rs/swc_common/errors/emitter/trait.Emitter.html), just like `IssueEmitter` if you want to read the issues.

Reply:
Thanks, might going to try later - at least this path doesn't seem to be explicitly required in next.js

Reply:
Maybe just store the list in the `IssueEmitter` and grab it from there